Sources/NIOCore/ByteBuffer-core.swift

```diff
@@ -675,7 +673,7 @@ public struct ByteBuffer {
     }
 
     /// Yields a buffer pointer containing this `ByteBuffer`'s readable bytes. You may hold a pointer to those bytes
-    /// even after the closure returned iff you model the lifetime of those bytes correctly using the `Unmanaged`
```

Comment:
"iff" means "if and only if" and is correct here

Reply:
🤦🏼 TIL, thank you!